Jenkinsfile

```diff
@@ -235,7 +235,9 @@ pipeline {
 
         stage('Test') {
             agent { label 'tester' }
-            when { environment name: "DONT_CREATE_TESTBEDS", value: null }
+            when { environment name: "DONT_CREATE_TESTBEDS", value: null;
+                   environment name: "NIGHTLY_BUILD", value: null
```

Comment:
Maybe we should add `NIGHTLY_BUILD` to other steps related to tests, such as `Deploy` or `Testenv provisioning`, so that only build stage is run?

Either way, I totally forgot about it, but as you can see, when `DONT_CREATE_TESTBEDS` is set only build stage is run. Maybe we should use it instead?